hash2curve/src/hash2field/expand_msg/xmd.rs

```diff
@@ -146,6 +145,35 @@ where
     }
 }
 
+/// Error type for [`ExpandMsgXmd`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ExpandMsgXmdError {
+    /// The domain separation tag is invalid because it is empty.
+    EmptyDst,
+    /// The domain separation tag is too long and needs to be hashed, but the hash function
+    /// selected has an output size too large (greater than `255`).
```

Comment:
This error message requires too much knowledge about `expand_message_xmd()`.
```suggestion
    /// The hash's output size must not be greater then `255`
    /// if the domain separation tag is longer than `255`.
```